packages/modules/devices/kostal/kostal_plenticore/inverter.py

```diff
@@ -32,14 +32,24 @@ def initialize(self) -> None:
         self.store = get_inverter_value_store(self.component_config.id)
         self.fault_state = FaultState(ComponentInfo.from_component_config(self.component_config))
         self.sim_counter = SimCounter(self.kwargs['device_id'], self.component_config.id, prefix="Wechselrichter")
+        self.fault_text = (
+            "Es kann keine PV Leistung aus dem Wechselrichter ausgelesen werden, "
```

Comment:
```suggestion
            "Es kann keine DC-Leistung aus dem Wechselrichter ausgelesen werden, "
```
PV-Leistung verbinden die meisten vermutlich mit dem Wert in der Variable power und wundern sich dann, warum es doch funkioniert.